src/Editor.vue

```diff
@@ -1,10 +1,13 @@
 <template>
-  <PrismEditor
-    v-model="stableCode"
-    :language="prismLang"
-    @change="updatePreview"
-    v-bind="editorProps"
-  />
+  <div>
+    <PrismEditor
+      v-model="stableCode"
+      :language="prismLang"
+      @change="updatePreview"
+      v-bind="editorProps"
+    />
+    <slot></slot>
```

Comment:
I would almost expect this to be the code displayed in the Editor.
Do you think it would be confusing to have a non-obvious slot here?

Reply:
maybe give it a name?

Reply:
You're right. This slot needs a name. I'll do it :) Thanks